dev/create_user.py

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright 2021 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import boto3
+import configparser
+from pathlib import Path
+import sys
+
+# Usage: python create_user.py $CORTEX_CLUSTER_NAME $CORTEX_ACCOUNT_ID $CORTEX_REGION
+
+cluster_name = sys.argv[1]
+account_id = sys.argv[2]
+cortex_region = sys.argv[3]
+
+with open("./dev/minimum_aws_policy.json", "r") as f:
```

Comment:
nit: consider adding `dir_path = os.path.dirname(os.path.realpath(__file__))` and using `f"{dir_path}/minimum_aws_policy.json"`, since this way, this script can be executed from any directory